### ISSUE TITLE
BIGINT returned as integer instead of string when used json_agg(to_jsonb())

### ISSUE THREAD

**Issue by weipinglim14** [brianc/node-postgres#2385]

**Comment by boromisp** [comment]
This is unfortunately expected, as the [driver simply uses `JSON.parse()`](https://node-postgres.com/features/types#uuid--json--jsonb) to parse `json` and `jsonb` values.

There is no good solution, you either loose precision, or performance and interoperability, by implementing a custom JSON parser that parses numbers as a [custom type](https://github.com/MikeMcl/big.js).

It's probably worth calling out in the documentation.

**Comment by weipinglim14** [comment]
Yes definitely. The workaround would be to use json_build_object and manually cast it as string in order for the id to return as string specifically.
